inc/classes/class-settings.php

```diff
@@ -194,8 +194,19 @@ public function wp_google_login_enable_registrationr() {
 		}
 		?>
 		<input type='hidden' name='wp_google_login_settings[registration_enabled]' value='0' <?php echo esc_attr( $disabled ); ?> >
-		<input type='checkbox' name='wp_google_login_settings[registration_enabled]' id="enable-registration" <?php echo esc_attr( checked( $registration_enabled ) ); ?> <?php echo esc_attr( $disabled ); ?> value='1'>
-		<p class="description"><?php esc_html_e( 'If user does not exist, WordPress will create a new user with the data provided by Google.' ); ?></p>
+		<label style='display:block;margin-top:6px;'><input type='checkbox' name='wp_google_login_settings[registration_enabled]' id="enable-registration" <?php echo esc_attr( checked( $registration_enabled ) ); ?> <?php echo esc_attr( $disabled ); ?> value='1'>
+			<?php esc_html_e( __( 'Create a new user account if it doesn’t exist already', 'login-with-google' ) ) ?>
+		</label>
+		<p class="description"><?php 
+			echo wp_kses_post( sprintf( 
+				'%1s <a target="_blank" href="%2s">%3s</a> %4s.', 
+				__( 'If this setting is checked, a new user will be created even if', 'login-with-google' ), 
+				'options-general.php',
+				__( 'membership setting', 'login-with-google' ),
+				__( 'is off', 'login-with-google' )
+			) );
```

Comment:
Breaking the strings will make it difficult to translate. So please keep the single string and use `wp_kses()` to allow only `a` tag and used attributes.

Reply:
Updated. Also corrected membership settings link for multisite.